### ISSUE TITLE
Python Script Restarts Hosting Services

### ISSUE THREAD

**Issue by millelog** [OSU-CS290-F16/final-project-cgrb-actf#10]
This is the main issue that I think it's having. We aren't able to do anything directly after the class is created because both the node server and the Nginx server are restarted by the python script... how should we fix this?

**Comment by Kirksten3** [comment]
How are they restarted? Does the script call service restart?

**Comment by Kirksten3** [comment]
Mind hopping in slack to discuss this?

**Comment by morgaalb** [comment]
Is the script restarting the Node server intentionally? If so, why?

**Comment by morgaalb** [comment]
This is a pretty big issue that we need to resolve. A script that is called by the user of a web page that restarts the web server... That seems like a huge design flaw.

I propose we decouple our app from the ACTF servers and run everyone on our own MySQL server. We should forget about all the host and port stuff, and just do a simple class creation and viewing app.

**Comment by Kirksten3** [comment]
Well the problem that we ran into was that the NGINX server needs to restart, as docker makes some config files for it, which nginx doesn't recognize unless it is restarted. We found a reload command which reloads the server but it kills all current workers and starts a new one. So it's kind of in the same boat still. I talked briefly to logan about the node problems and he said he would look into it for the time being. But I will help any way I can. 

If getting our own DBs works then I'm game for that. We can setup production environments so we won't have to change much code if wanted too.
